assets/javascripts/discourse/components/discourse-post-event/index.gjs

```diff
@@ -0,0 +1,198 @@
+import Component from "@glimmer/component";
```

Comment:
1:1 migration, most notable change is that message bus is not setup in this component instead of the initializer